examples/rosenbrock_lcb.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+import numpy as np
+
+# Import ConfigSpace and different types of parameters
+from smac.configspace import ConfigurationSpace
+from ConfigSpace.hyperparameters import UniformFloatHyperparameter
+
+# Import SMAC-utilities
+from smac.scenario.scenario import Scenario
+from smac.facade.smac_facade import SMAC
+from smac.optimizer.acquisition import LCB
+from smac.initial_design.latin_hypercube_design import LHDesign
+from smac.runhistory.runhistory2epm import RunHistory2EPM4InvScaledCost
+
+def rosenbrock_2d(x):
+    """ The 2 dimensional Rosenbrock function as a toy model
+    The Rosenbrock function is well know in the optimization community and
+    often serves as a toy problem. It can be defined for arbitrary
+    dimensions. The minimium is always at x_i = 1 with a function value of
+    zero. All input parameters are continuous. The search domain for
+    all x's is the interval [-5, 5].
```

Comment:
I think more common is the interval [-5, 10]

Reply:
Thanks, just changed that.